core/src/edu/asu/c3simulator/screens/CompanyPanel.java

```diff
@@ -0,0 +1,284 @@
+		
+		labels.addActor(businessNameLabel);
+		labels.addActor(businessNetWorthLabel);
+		labels.addActor(businessSellingPriceLabel);
+		
+		information.addActor(businessName);
+		information.addActor(businessNetWorth);
+		information.addActor(businessSellingPrice);
+		
+		labels.align(Align.left);
+		information.align(Align.left);
+		labels.space(5);
+		information.space(5);
+		
+		businessInfo.add(labels).spaceRight(10);
+		businessInfo.add(information).expand().fill().left();
+		// businessInfo.debug();
+		
+		return businessInfo;
+	}
+	
+	/**
+	 * This method should update the information section with the appropriate info for the
+	 * selected business.
+	 */
+	private void updateBusinessInfo()
+	{
+		// TODO add items to the list
+		// TODO test how to manipulate it
+	}
+	
+	/**
+	 * This method creates the buttons that give the option to rename or sell the selected
+	 * business.
+	 * 
+	 * @return The GUI component created.
+	 */
+	private Actor createBusinessButtons()
+	{
+		VerticalGroup businessButtons = new VerticalGroup();
+		
+		// TODO add rename and sell button created by teammates
+		
+		TextButton renameButton = new TextButton("Rename", skin);
+		TextButton sellButton = new TextButton("Sell", skin);
+		
+		businessButtons.addActor(renameButton);
+		businessButtons.addActor(sellButton);
+		businessButtons.align(Align.center);
+		businessButtons.space(10);
+		
+		return businessButtons;
+	}
+	
+	/**
+	 * This method creates the GUI component holding graph that represents the selected
+	 * business.
+	 * 
+	 * @return The GUI component created.
+	 */
+	private Actor createBusinessGraph()
+	{
+		VerticalGroup businessGraph = new VerticalGroup();
+		
+		// TODO create graph and display it
+		
+		Label graphLabel = new Label("Net Profit vs. Time", skin);
+		
+		FileHandle iconLocation = Gdx.files
+				.internal("images/placeholder-advisor-icon.png");
+		Texture iconTexture = new Texture(iconLocation);
+		Image advisorIcon = new Image(iconTexture);
+		
+		updateBusinessGraph();
+		
+		businessGraph.addActor(graphLabel);
+		businessGraph.addActor(advisorIcon);
+		businessGraph.align(Align.center);
+		
+		return businessGraph;
+	}
+	
+	/**
+	 * This method update the graph based on the selected business.
+	 */
+	private void updateBusinessGraph()
+	{
+		// TODO add items to the list
+		// TODO test how to manipulate it
+	}
+	
+	@Override
+	public void render(float delta)
+	{
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+		stage.act(Gdx.graphics.getDeltaTime());
+		stage.draw();
+	}
+	
+	@Override
+	public void resize(int width, int height)
+	{
+		stage.getViewport().update(width, height);
+	}
+	
+	@Override
+	public void show()
+	{
+		Gdx.input.setInputProcessor(stage);
+		
+	}
+	
+	@Override
+	public void hide()
+	{
+		Gdx.input.setInputProcessor(null);
+		
+	}
+	
+	@Override
+	public void pause()
+	{
+		// TODO Auto-generated method stub
+		// throw new UnsupportedOperationException("The method is not implemented yet.");
```

Comment:
If "do nothing" has been verified as the desired functionality of this method, then replace these two lines with a comment that says "// do nothing"

Otherwise, add a TODO, or add the functionality, or keep the exception (although this will lead to other issues, as pause() seems to be in use)